eq-author/src/App/metadata/MetadataPage.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { flowRight } from "lodash";
+import styled from "styled-components";
+
+import Error from "components/Error";
+import Loading from "components/Loading";
+import Button from "components/buttons/Button";
+
+import ScrollPane from "components/ScrollPane";
+import Header from "components/EditorLayout/Header";
+import { Grid } from "components/Grid";
+import MainCanvas from "components/MainCanvas";
+
+import MetadataTable from "./MetadataTable";
+
+import withCreateMetadata from "./withCreateMetadata";
+import withDeleteMetadata from "./withDeleteMetadata";
+import withUpdateMetadata from "./withUpdateMetadata";
+import GetMetadataQuery from "./GetMetadataQuery";
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+`;
+
+const StyledGrid = styled(Grid)`
+  overflow: hidden;
+`;
+
+const StyledMainCanvas = styled(MainCanvas)`
+  padding: 0 0.5em 0 1em;
+  max-width: 80em;
+`;
+
+const Toolbar = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  margin: 1em auto;
+  padding: 0 1em;
+  width: 100%;
+  max-width: 80em;
+`;
+
+export class UnwrappedMetadataPage extends Component {
+  renderContent = ({ loading, error, data: { questionnaire } }) => {
+    const { onAddMetadata, onDeleteMetadata, onUpdateMetadata } = this.props;
+
+    if (loading) {
+      return <Loading height="100%">Questionnaire metadata loading…</Loading>;
+    }
+
+    if (error) {
+      return <Error>Oops! Something went wrong</Error>;
+    }
+
+    if (!questionnaire) {
+      return <Error>Oops! Questionnaire could not be found</Error>;
+    }
+
+    return (
+      <Container>
+        <Header title="Metadata" />
+        <Toolbar>
+          <Button onClick={() => onAddMetadata(questionnaire.id)}>
+            Add metadata
+          </Button>
+        </Toolbar>
+        <StyledGrid>
+          <ScrollPane permanentScrollBar data-test="metadata-modal-content">
+            <StyledMainCanvas>
+              <MetadataTable
+                metadata={questionnaire.metadata}
+                questionnaireId={questionnaire.id}
+                onAdd={onAddMetadata}
+                onDelete={onDeleteMetadata}
+                onUpdate={onUpdateMetadata}
+              />
```

Comment:
![Screenshot 2019-08-27 at 08 10 02](https://user-images.githubusercontent.com/35268220/63748885-26a33c80-c8a2-11e9-8dce-abae8c004d03.png)
Without the button here the empty state on the table looks kinda odd. Might be worth checking in with Joe and see if its worth putting a placeholder in here?

Reply:
After speaking to Joe, a No Metadata page is displayed instead